Assets/Scripts/GraphService.cs

```diff
@@ -10,11 +10,13 @@ namespace Assets.Scripts
   public class GraphService : MonoBehaviour
   {
     bool _labelVisibility;
-    public EdgeGameObjectFactory edgeGameObjectFactory;
-    public LineFactory lineFactory;
+    public GameObject labelPrefab;
+    public LineFactory lineFactory = new LineFactory();
```

Comment:
DI needed asap xd

Reply:
Yup. I am slowly getting the project ready by removing as many MonoBehaviors from scripts as possible. Hopefully it will turn out nicely